src/main/resources/messages_en.properties

```diff
@@ -72,10 +72,10 @@ requisition.error.validation.mustBeDisplayed={0} must be displayed.
 requisition.error.validation.requestedQuantityExplanationRequired=Missing value in requested quantity explanation column
 requisition.error.validation.fieldIsTooLong={0} is too long. The maximum length is {1}
 requisition.error.validation.cannotUpdateRequisition=The requisition could not be updated because it is not initiated, submitted or authorized.
-requisition.error.validation.dateModifiedMismatch=Date modified of the passed requisition does not match the last modification date - concurrent changes possible.
+requisition.error.validation.dateModifiedMismatch=Date modified of the past requisition does not match the last modification date - concurrent changes possible.
```

Comment:
Perhaps 'the requisition you attempted to sync', since the sync failed? I guess 'synced' works I guess, although it might make people think the sync was successful?